.tekton/go-lint-pipelinerun.yaml

```
@@ -9,6 +9,11 @@ metadata:
     pipelinesascode.tekton.dev/task: "git-clone"
     pipelinesascode.tekton.dev/task-1: "golangci-lint"
     pipelinesascode.tekton.dev/max-keep-runs: "5"
+    # Expression to check for .go files inside pkg & steps directory
+    # "pkg/*/*.go".pathChanged() || "steps/*/*.go".pathChanged() || "steps/*.go".pathChanged()
```

you can remove the this line
```suggestion
    # "pkg/*/*.go".pathChanged() || "steps/*/*.go".pathChanged() || "steps/*.go".pathChanged()
```